### DISCUSSION TITLE
关于重置版的测试

### DISCUSSION THREAD

**Discussion by ALTaleX531** [ALTaleX531/TranslucentFlyouts#22] [Announcements]
代码已被重构，测试完毕时将会发布新版本

**Comment by ALTaleX531** [comment]
[TranslucentFlyouts-Remake.zip](https://github.com/ALTaleX531/TranslucentFlyouts/files/11010587/TranslucentFlyouts-Remake.zip)
运行install.bat安装，添加自启动项
运行uninstall.bat卸载，用户注销后即可
run.bat负责以Rundll32为载体，启动TranslucentFlyouts，在运行后会自动挂起自身，避免无意义的资源浪费
![image](https://user-images.githubusercontent.com/105535231/226165898-691e2d52-5583-47d5-8a0b-ee202e3c17de.png)
如果你想中途关闭TranslucentFlyouts怎么办？
![image](https://user-images.githubusercontent.com/105535231/226165834-4a9fc417-7bae-448d-9bd2-31915ef28b1e.png)
找到暂停的Rundll32并恢复即可自动退出

**Comment by ALTaleX531** [comment]
有需求或者bug反馈，以及相关建议可以在discussion下各板块提出！

**Comment by shamodeyu** [comment]
1. 有蓝色边框，可以设置不加框？
![2023-03-20 10 37 52](https://user-images.githubusercontent.com/123850195/226236839-821709ef-9446-4f6b-8974-dfeefa158212.png)
![2023-03-20 10 38 23](https://user-images.githubusercontent.com/123850195/226236843-c84301fd-d2f8-4e50-a7fa-8d6235cd76aa.png)
2. 透明度过高，背景呈现斑驳样，不美观，尤其是explore内，白色背景情况下，更不美观。
![2023-03-20 10 43 03](https://user-images.githubusercontent.com/123850195/226236888-a85a8925-c3d1-4b64-a7d5-9886c9ce4d8f.png)
![2023-03-20 10 41 59](https://user-images.githubusercontent.com/123850195/226236894-1cbc18c9-ef9f-4c0c-8814-305ea4fc13da.png)
3. 未启用时，右键的图标颜色正常；启用后，图标颜色变色，还有白色背景。
![2023-03-20 10 45 49](https://user-images.githubusercontent.com/123850195/226236951-09b0f271-972b-496b-a153-be03b0489171.png)
![2023-03-20 10 39 15](https://user-images.githubusercontent.com/123850195/226236973-e989ba67-04db-45a5-866b-abe324108489.png)
4.建议：
亚克力效果随背景变换，有其特点。但是对于白色背景，如explore内，效果不佳。希望可以一直类似桌面蓝色背景时的效果（如Mica？）
![2023-03-20 10 40 42](https://user-images.githubusercontent.com/123850195/226237050-615d70a4-b0f2-4d8a-89cb-880966766ea0.png)
![2023-03-20 10 40 17](https://user-images.githubusercontent.com/123850195/226237059-6ccaac59-ab47-4f3a-8aa4-ba506acdf36e.png)
![2023-03-20 10 42 21](https://user-images.githubusercontent.com/123850195/226237073-9fff43f1-5bcf-4e06-bfa0-6b6fcd10dfc5.png)
5. 建议：
右键单击任务栏，其弹窗背景是固定的，能不能改为蓝色背景效果？
![2023-03-20 10 48 08](https://user-images.githubusercontent.com/123850195/226237106-42ece2d3-a909-4758-a069-ddec35380f15.png)

**Reply by ALTaleX531** [reply]
由于系统问题，win10是不支持云母的，但是颜色这个东西比较好解决，至于图标异常，那是因为那些软件使用的图标不是32位的位图，这个解决起来比较麻烦

**Comment by AquaHarp** [comment]
有概率触发这个bug
有使用explorerpatcher(版本22621.1555.55.1)和open-shell(版本4.4.189)
![image](https://user-images.githubusercontent.com/25244319/235338729-f8f09d91-036f-45f9-b2db-caa58ebc2f2b.png)
![image](https://user-images.githubusercontent.com/25244319/235338734-e7fbe2d0-356a-41ff-a912-10c7f3813947.png)
![image](https://user-images.githubusercontent.com/25244319/235338834-f34a775a-0912-43ae-8d16-aa8851249f43.png)

**Reply by ALTaleX531** [reply]
我现在制作的版本应该没有此类问题了，应该2.0的发布能解决你的问题（不出意外的话

**Comment by ice051128** [comment]
This test version does absolutely nothing on windows 10, is it not supported?

![image](https://github.com/ALTaleX531/TranslucentFlyouts/assets/49018832/611229f0-2ba3-4949-90b4-59f5a0d978f2)

**Reply by ALTaleX531** [reply]
There is a syntax error in the install.bat, so the install.bat did nothing. you can try to replace these bat files with new one...
[bat.zip](https://github.com/ALTaleX531/TranslucentFlyouts/files/11933386/bat.zip)

**Comment by shamodeyu** [comment]
终于等到7月了，开发完成了没？翘首以盼啊！

**Reply by ALTaleX531** [reply]
还有配置文件的读取，界面的编写没有完成，可能得等个几天，新加了4种效果，修复了冲突性问题和稳定性问题等一大堆问题，内容还是比较多的哈

**Comment by ALTaleX531** [comment]
![屏幕截图 2023-07-02](https://github.com/ALTaleX531/TranslucentFlyouts/assets/105535231/5a4181fb-ba24-4b9e-9816-7bf93d2f5368)
windows 11的云母

**Comment by ALTaleX531** [comment]
已经更新了readme和发布了v2.0.0-alpha，由于时间比较仓促，没来得及写UI程序，得需要自己手动修改注册表了，这部分详见readme

**Reply by shamodeyu** [reply]
WIn10 2004版本，安装后出现以下提示
![2023-07-17 08 09 23](https://github.com/ALTaleX531/TranslucentFlyouts/assets/123850195/4d7afd44-4ac9-458c-924e-dfc4040d1113)

另在注册表中修改参数，似乎无法修改显示效果。

现在只好装回之前的测试版，期待UI程序完成的版本。

**Reply by ALTaleX531** [reply]
已经确认为一个bug，刚刚修复了，过会发

**Reply by shamodeyu** [reply]
部分二级菜单是黑色的，鼠标移过去才会显示。
![2023-07-18 09 37 29](https://github.com/ALTaleX531/TranslucentFlyouts/assets/123850195/b7ee996a-fa16-442b-afa4-0060fecbbb6f)
![2023-07-18 09 38 03](https://github.com/ALTaleX531/TranslucentFlyouts/assets/123850195/ffe530fe-899b-4dbb-add9-9821ea394a23)

**Reply by ALTaleX531** [reply]
能否详细说明是如何触发的，我这里没有这样的问题，你显示的那个菜单还有异常的浅色边，推测是被杀软干扰了

**Reply by shamodeyu** [reply]
1. 安装之后，按要求注销，再次登录即存在该二级菜单黑色的情况。
2.注册表中没有 HKEY_CURRENT_USER\SOFTWARE\TranslucentFlyouts\ ，无法尝试各种定制

**Reply by shamodeyu** [reply]
重复安装和卸载后，再安装，二级菜单没有黑色了。
不过菜单外面有一圈蓝色的线。
![2023-07-21 16 27 04](https://github.com/ALTaleX531/TranslucentFlyouts/assets/123850195/027cb354-2cc0-4496-9031-7b47c1921626)
注册表中仍然没有 HKEY_CURRENT_USER\SOFTWARE\TranslucentFlyouts\ ，无法尝试各种定制。

**Reply by ALTaleX531** [reply]
我明白了，可能是之前的版本没有卸载干净，因为只有之前的重置版才有默认有蓝色的线，如果注册表没有这个项，需要你手动进行创建

**Reply by shamodeyu** [reply]
现在有白色边框
![2023-08-02 08 02 27](https://github.com/ALTaleX531/TranslucentFlyouts/assets/123850195/1cb5fa81-30b2-4fe9-bf86-90f9a2b6142d)
